modules/sd-vs-ocp.adoc

```diff
@@ -16,24 +16,42 @@ Review the following differences between {product-title} and a standard installa
+endif::openshift-dedicated[]
+ifdef::openshift-rosa[]
+{product-title} is installed through {cluster-manager-first} or the ROSA CLI (`rosa`) and in a standardized way that is optimized for performance, scalability, and security.
+endif::openshift-rosa[]
 
 |Customers can choose their computing resources.
-|{product-title} is hosted and managed in a public cloud (Amazon Web Services or Google Cloud Platform) either owned by Red Hat or provided by the customer.
+|
+ifdef::openshift-dedicated[]
+{product-title} is hosted and managed in a public cloud (Amazon Web Services or Google Cloud Platform) either owned by Red Hat or provided by the customer.
+endif::openshift-dedicated[]
+ifdef::openshift-rosa[]
+{product-title} is hosted and managed in a public cloud (Amazon Web Services) provided by the customer.
+endif::openshift-rosa[]
 
 |Customers have top-level administrative access to the infrastructure.
-|Customers have a built-in administrator group, though the top-level administration access is available when cloud accounts are provided by the customer.
+|
+ifdef::openshift-dedicated[]
+Customers have a built-in administrator group (`dedicated-admin`), though the top-level administration access is available when cloud accounts are provided by the customer.
+endif::openshift-dedicated[]
+ifdef::openshift-rosa[]
+Customers have a built-in administrator group (`dedicated-admin`), though the top-level administration access is available.
+endif::openshift-rosa[]
 
 |Customers can use all supported features and configuration settings available in {OCP}.
-|Some {OCP} features and configuration settings might not be available or changeable in {product-title} .
+|Some {OCP} features and configuration settings might not be available or changeable in {product-title}.
 
-|You set up control plane components such as the API server and etcd on machines that get the `control` role. You can modify the control plane components, but keep in mind that you are responsible for backing up, restoring, and making control plane data highly available.
+|You set up control plane components such as the API server and etcd on machines that get the `control` role. You can modify the control plane components, but are responsible for backing up, restoring, and making control plane data highly available.
 |Red Hat sets up the control plane and manages the control plane components for you. The control plane is highly available.
 
 |You are responsible for updating the underlying infrastructure for the control plane and worker nodes. You can use the OpenShift web console to update {OCP} versions.
-|Red Hat automatically notifies the customer when updates are available. You can manually or automatically schedule upgrades in {cluster-manager-first}.
+|Red Hat automatically notifies the customer when updates are available. You can manually or automatically schedule upgrades in {cluster-manager}.
```

Comment:
Update is the preferred term per our [term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update):
```suggestion
|Red Hat automatically notifies the customer when updates are available. You can manually or automatically schedule updates in {cluster-manager}.
```